parcels/examples/example_dask_chunk_OCMs.py

```diff
@@ -226,9 +226,8 @@ def test_pop(mode, chunk_mode):
     if chunk_mode is False:
         assert (len(field_set.U.grid.load_chunk) == 1)
     elif chunk_mode == 'auto':
-        assert (len(field_set.U.grid.load_chunk) != 1)
+        assert (len(field_set.U.grid.load_chunk) == 1)
```

Comment:
To be honest; not sure why this unit test had to be changed now...

Reply:
if you had to change that, then something was wrong - what was the error message here ? what was the actual value of `field_set.U.grid.load_chunk` ?

Reply:
It is now `[2]`, while it is `[2 0]` in the current `master` branch. Not sure why, but see also https://github.com/OceanParcels/parcels/pull/936#discussion_r509144766 below..

Reply:
well, did you change the test data in their layout ?

Reply:
No, but the PR did change the size of the FieldSet because we don't add an extra layer to the bottom of the Fields anymore. That could cause this change?